src/actions/actionProperties.tsx

```diff
@@ -622,3 +626,99 @@ export const actionChangeSharpness = register({
     </fieldset>
   ),
 });
+
+export const actionChangeDecorator = register({
+  name: "changeDecorator",
+  perform: (elements, appState, value) => {
```

Comment:
can we track it somewhere? Here is the only place that the magic happens.. right?

```
trackEvent(EVENT_CHANGE, "decorator start",  "arrow");
trackEvent(EVENT_CHANGE, "decorator end",  "null");
trackEvent(EVENT_CHANGE, "decorator end",  "arrow");
```

Reference: [`trackEvent`](https://github.com/excalidraw/excalidraw/blob/master/src/analytics.ts#L13-L20)

Example: [`actionProperties.tsx`](https://github.com/excalidraw/excalidraw/blob/master/src/actions/actionProperties.tsx#L127-L133)